libs/icu/release-72-1/wl-env-repo.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+if [[ $1 == "--unset" ]]
+then
+    unset WASMLABS_REPO
+    unset WASMLABS_REPO_NAME
+    return
+fi
+
+export WASMLABS_REPO=https://github.com/unicode-org/icu.git
+export WASMLABS_REPO_NAME=icu
+export WASMLABS_REPO_BRANCH=release-72-1
```

Comment:
Needed to decouple branch names in the repo with whatever structure we want to have in WLR.

Reply:
As explained in our chat, this was needed to void a bigger refactor in all the other scripts :-)